server/clearbit/client.js

```diff
@@ -45,6 +87,6 @@ export default class ClearbitClient {
   prospect(params) {
     this.metric("clearbit.prospect");
     this.log("clearbit.prospect", JSON.stringify(params));
-    return this.client.Prospector.search(params);
```

Comment:
Because "Clearbit client does not handle arrays properly in querystring" : 
https://github.com/hull-ships/hull-clearbit/commit/c0530e4fd16a60add57a77262f8884f0731bfec9